src/__test__/header.test.js

```diff
@@ -0,0 +1,30 @@
+import { screen, render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import '@testing-library/jest-dom';
+import Header from '../coponents/header/Header';
+
+const HeaderForTesting = (
+  <BrowserRouter>
+    <Header />
+  </BrowserRouter>
+);
+describe('Testing Header Component', () => {
+  render(HeaderForTesting);
```

Comment:
- [ ] Great job writing test for your components, however, its one of the project requirement to write test for pure redux functions kindly implement that 👍.

![Screen Shot 2022-10-14 at 10 15 28 AM](https://user-images.githubusercontent.com/59416136/195785653-69728f2b-1720-4ef9-8b07-1944c514a463.png)

Reply:
Hello @Wes-Isaac, thank you for your constructive feedback. The reason why I didn't write a test for the pure function is, I don't have a pure function in my redux code. I have only one thunk function. I would be happy if you give some context by looking at my code.